client/src/pages/ManageSteps/dummyData.js

```diff
@@ -0,0 +1,14 @@
+export default [
```

Comment:
Instead of dummy data, you could just use the steps context which fetches all the steps for you from the server, including the stage, title and order, and then tidy up the front end accordingly. Speak to Ramy in case he wants a dedicated api call for this